pkgs/development/misc/avr-gcc-with-avr-libc/default.nix

```diff
@@ -21,7 +21,7 @@ stdenv.mkDerivation {
   };
 
   srcAVRLibc = fetchurl {
-    url = http://download.savannah.gnu.org/releases/avr-libc/avr-libc-1.7.1.tar.bz2;
+    url = http://ftp.igh.cnrs.fr/pub/nongnu/avr-libc/old-releases/avr-libc-1.7.1.tar.bz2;
```

Comment:
I tested yesterday and somehow the result was not ok.
I may have been too prompt.
Good, I'll adapt.